README.md

```diff
@@ -1,2 +1,55 @@
-# embedding_images_videos
-First microverse project html_css
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Embedding images and videos
+
+> Collaborative project course html and css course Marcelo and Julián
+
+![screenshot](./app_screenshot.png)
+
+First project main curriculum
+
+## Built With
+
+- HTML and CSS
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+## Authors
+
+👤 **Julián**
+
+- GitHub: [@julian3493](https://github.com/julian3493)
+- Twitter: [@JulianR16893833](https://twitter.com/JulianR16893833)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/juli%C3%A1n-ricardo-ramos-arevalo-3868ba135/)
+
+👤 **Author2**
+
+- GitHub: [@marcelomaidden](https://github.com/marcelomaidden)
+- Twitter: [@marcelomaidden](https://twitter.com/marcelomaidden)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/marcelo-fernandes-de-ara%C3%BAjo-56700a171/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
```

Comment:
You have to fix this section too. You're to acknowledge anyone whose resource(s) were used in your project (e.g. youtube).

Reply:
We acknowledged W3Schools where we found information on how to style elements and Youtube where we found images and layout to serve as base to our project.